services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy

```diff
@@ -1145,6 +1147,70 @@ mode with existing flows and hold flows of different table-mode types"() {
         northbound.deleteLinkProps(northbound.getAllLinkProps())
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2932")
+    def "System does not allow ot enable the multiTable mode on an unsupported switch"(){
+        given: "Unsupported switch"
+        def sw = topology.activeSwitches.find { !it.features.contains(SwitchFeature.MULTI_TABLE) }
+        assumeTrue("Unable to find required switch", sw as boolean)
+
+        when: "Try to enable the multiTable mode on the switch"
+        northbound.updateSwitchProperties(sw.dpId, northbound.getSwitchProperties(sw.dpId).tap {
+            it.multiTable = true
+        })
+
+        then: "Human readable error is returned"
+        def exc = thrown(HttpClientErrorException)
+        exc.rawStatusCode == 404
+        //TODO(andriidovhan) fix error message when 2932 is fixed
+        exc.responseBodyAsString.to(MessageError).errorMessage == "Switch doesn't support multiTable"
+    }
+
+    def "System connects a new switch with disabled multiTable mode when the switch does not support that mode"() {
+        given: "Unsupported switch"
+        def sw = topology.activeSwitches.find { !it.features.contains(SwitchFeature.MULTI_TABLE) }
+        assumeTrue("Unable to find required switch", sw as boolean)
+        def initConf = northbound.getKildaConfiguration()
+        !initConf.useMultiTable && northbound.updateKildaConfiguration(northbound.getKildaConfiguration().tap {
+            it.useMultiTable = true
```

Comment:
It is an important step of the test case, I believe state of global config should be explicitly described as a precondition

Reply:
added